src/DistributedMetadata/MetadataService.cpp

```diff
@@ -309,4 +312,36 @@ void MetadataService::startup()
 
     postStartup();
 }
+
+bool MetadataService::tryAppendDWal(const DWAL::Record & record, const DWAL::KafkaWALContext & ctx, bool is_support_throw) const
```

Comment:
Let's remove / clean up all retries in application (daisy), since we already setup the retries in librdkafka. `message_send_max_retries` in config.xml

Reply:
I ignored Kafka's own retrial logic, which should be deleted